pkgs/by-name/di/disnix-web-service/package.nix

```diff
@@ -28,7 +40,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   env = {
     PREFIX = "\${env.out}";
```

Comment:
I think we should prefer using `${placeholder "out"}` here, though I might be wrong, in which case you may dismiss.

Reply:
never heard of placeholder before, but it makes sense